docs/basics/acct-team-mgmt/private-device-mgmt.md

```diff
@@ -91,6 +91,25 @@ Update this setting for each account email to preserve store and payment account
 `account2@mydomain.com`  
 :::
 
+## Retain specific filepaths/folders on the device (Android)
+
+You have the option to preserve specific filepaths or folders on your private Android devices between sessions. 
+This feature ensures that selected data remains intact during the device cleaning process.
+
+- Filepaths or folders associated with specific applications can be retained.
+- This is especially useful for retaining data that is crucial for continuous testing or for maintaining certain app states.
+
+By default, the following folders are not cleaned on your private Android devices, ensuring that their contents are retained between sessions (iOS/Android):
+- Documents
+- Pictures
+- Downloads
+- DCIM
+
+:::important
+`/data/com.wavelink.velocity/files`  
```

Comment:
should we mention a side note also that also for tmp storage only data inside  /data/local/tmp will be retained so the path should start  with `/data/local/tmp` ?

Reply:
yup! adding it!